### ISSUE TITLE
Website issues

### ISSUE THREAD

**Issue by ghost** [tsparticles/tsparticles#4897]
### What happened?

Hello Sir M-m-m-m-matteo Bruno!

It seems that some presets are not working?. when I copy paste confetti and fireworks presets, for example

https://particles.js.org/samples/presets/confetti.html
https://particles.js.org/samples/presets/confetti.html
https://codepen.io/matteobruni/pen/abJQrbK

They used to work until 2.6 or 2.7.Maybe 2.71 or 2.8 changed some configs?
how to correct?

Also, is the dependency graph updated? It seems recent changes were made to tsparticles-slim?
Also, could you please update the dependency graph style? -> Arrows are so close, I can't see which one goes to tsparticles or tsparticles-slim. A text version would also be appreciated in the docs? Or already exists?

Thank you!!!!!

### tsParticles Version

2.8

### Which library are you using?

Web Components (web-particles)

### tsParticles Configuration

```json
VANILLA and WORDPRESS
```

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
> ### Contact Details
> _No response_
> 
> ### What happened?
> Hello Sir M-m-m-m-matteo Bruno!
> 
> It seems that some presets are not working?. when I copy paste confetti and fireworks presets, for example
> 
> https://particles.js.org/samples/presets/confetti.html https://particles.js.org/samples/presets/confetti.html https://codepen.io/matteobruni/pen/abJQrbK
> 
> They used to work until 2.6 or 2.7.Maybe 2.71 or 2.8 changed some configs? how to correct?
> 
> Also, is the dependency graph updated? It seems recent changes were made to tsparticles-slim? Also, could you please update the dependency graph style? -> Arrows are so close, I can't see which one goes to tsparticles or tsparticles-slim. A text version would also be appreciated in the docs? Or already exists?
> 
> Thank you!!!!!
> 
> ### tsParticles Version
> 2.8
> 
> ### Which library are you using?
> Web Components (web-particles)
> 
> ### tsParticles Configuration
> ```json
> VANILLA and WORDPRESS
> ```
> 
> ### What browsers are you seeing the problem on?
> Firefox
> 
> ### Relevant log output
> _No response_
> 
> ### Code of Conduct
> * [x]  I agree to follow this project's Code of Conduct

In the `confetti` and `fireworks` pages there were a missing plugin in both pages, `motion` for confetti and `sounds` for fireworks.

I'm fixing that, it should be available soon.

The dependency graphs are going to be reviewed for v3 (PR #4874). You can check the dependencies [here](https://github.com/matteobruni/tsparticles/blob/873d73df8b32abcc6ae4dcb042ae1a0b3ad2c1fc/bundles/slim/package.dist.json#L73) for `tsparticles-slim` and [here](https://github.com/matteobruni/tsparticles/blob/873d73df8b32abcc6ae4dcb042ae1a0b3ad2c1fc/bundles/full/package.dist.json#L73) for `tsparticles`

**Comment by ghost** [comment]
I can't get the particles to be emitted in 2 canvas, despite adding the new sound and motion modules.
These are old animations. One value of their objects must not be working anymore due to new updates,   or I am missing one dependency or another and can't figure it out what it is.
But it's okay... I am going to wait for the updated dependency graphs, the updated demos, presets, and please a updated animation creator on the website.
From there, I am going to export the updated presets, and merge with my old ones.
Thank you !!!!! and have a nice work ahead

**Comment by ghost** [comment]
For the record - (dual confetti, not emitted since 2.7 or 2.8 update - used to work until 2.6):

tsParticles-Slim

External Pause

Base

Emitters

Circle
Star

Color
Out modes
Roll
Size
Tilt
Wobble

{
	"autoPlay": true,
	"fullScreen": {
		"enable": true,
		"zIndex": 0
	},
	"detectRetina": true,
	"fpsLimit": 60,
	"interactivity": {
		"detectsOn": "window",
		"events": {
			"onClick": {
				"enable": true,
				"mode": "pause"
			},
			"resize": true
		}
	},
	"motion": {
		"disable": false,
		"reduce": {
			"factor": 4,
			"value": true
		}
	},
	"emitters": [
		{
			"rate": {
				"quantity": 5,
				"delay": 0.15
			},
			"particles": {
				"move": {
					"direction": "top-right",
					"outModes": {
						"top": "none",
						"left": "none",
						"default": "destroy"
					}
				}
			},
			"position": {
				"x": 0,
				"y": 30
			}
		},
		{
			"rate": {
				"quantity": 5,
				"delay": 0.15
			},
			"particles": {
				"move": {
					"direction": "top-left",
					"outModes": {
						"top": "none",
						"right": "none",
						"default": "destroy"
					}
				}
			},
			"position": {
				"x": 100,
				"y": 30
			}
		}
	],
	"pauseOnOutsideViewport": true,
	"particles": {
		"color": {
			"value": [
				"#ffffff",
				"#c29d0a"
			]
		},
		"move": {
			"decay": 0.05,
			"direction": "top",
			"enable": true,
			"gravity": {
				"enable": true
			},
			"outModes": {
				"top": "none",
				"default": "destroy"
			},
			"speed": {
				"min": 10,
				"max": 50
			}
		},
		"number": {
			"value": 0
		},
		"opacity": {
			"value": 1
		},
		"roll": {
			"darken": {
				"enable": true,
				"value": 25
			},
			"enable": true,
			"speed": {
				"min": 5,
				"max": 15
			}
		},
		"rotate": {
			"value": {
				"min": 0,
				"max": 360
			},
			"animation": {
				"enable": true,
				"speed": 30
			},
			"direction": "random"
		},
		"shape": {
			"type": [
				"circle",
				"star"
			]
		},
		"size": {
			"value": {
				"min": 0,
				"max": 2
			},
			"animation": {
				"count": 1,
				"enable": true,
				"speed": 16,
				"startValue": "min",
				"sync": true
			}
		},
		"tilt": {
			"direction": "random",
			"enable": true,
			"value": {
				"min": 0,
				"max": 360
			},
			"animation": {
				"enable": true,
				"speed": 30
			}
		},
		"wobble": {
			"distance": 30,
			"enable": true,
			"speed": {
				"min": -7,
				"max": 7
			}
		}
	}
}

**Comment by matteobruni** [comment]
You can see a sample using your config here: https://codepen.io/matteobruni/pen/xxJJjob?editors=0010

The plugins loaded are these:

```
  await loadSlim(tsParticles); // includes move-base, shape-circle, shape-star, updater-color, updater-out-modes, updater-size, and others, but based on your config only these are needed. You can load also updater-opacity if you want, but slim already contains it
  await loadExternalPauseInteraction(tsParticles);
  await loadEmittersPlugin(tsParticles);
  await loadMotionPlugin(tsParticles);
  await loadRollUpdater(tsParticles);
  await loadTiltUpdater(tsParticles);
  await loadWobbleUpdater(tsParticles);
```

**Comment by matteobruni** [comment]
The linked PR updates [tsparticles-slim](https://github.com/matteobruni/tsparticles/tree/main/bundles/slim#readme) and [tsparticles](https://github.com/matteobruni/tsparticles/tree/main/bundles/full#readme) readme files

**Comment by ghost** [comment]
Sir
I appreciate your every effort.
Could you please confirm that your sample ( https://codepen.io/matteobruni/pen/xxJJjob?editors=0010 ) works with the tsParticles block in WordPress?

I load the plugins you mentioned in the WordPress block, but it still doesn't work.
Which is interesting nonetheless, we seem to be discovering another possibility of improvement, and that is...

I tested some possible scenarios with the tsParticles block in WordPress (1- I loaded the plugins I used to load with my sample; 2- I load the plugins you mentioned with my sample; 3- I load the plugins I used to load with your sample; 4- I load the plugins you mention with your sample)

Your sample+plugins work in the WordPress editor through the tsParticles block, but it doesn't load on the front-end of the website.
This situation only loads correctly in the frontend when I choose "Load tsParticles" instead of "Load tsParticles Slim".

And there is something happening when choosing "Load Motion" that doesn't happen when choosing other options, but I didn't an extensive test to see if only "Load Motion" is producing a bug. And that is: after saving and reloading the page, the tsParticles block becomes broken - (because I marked the option "Load Motion". If I never choose "Load Motion", when saving and reloading everything, the block works and reloads correctly.

I hope these investigations and continued feedback helps your work.
Sincerely,

**Comment by matteobruni** [comment]
> Sir I appreciate your every effort. Could you please confirm that your sample ( https://codepen.io/matteobruni/pen/xxJJjob?editors=0010 ) works with the tsParticles block in WordPress?
> 
> I load the plugins you mentioned in the WordPress block, but it still doesn't work. Which is interesting nonetheless, we seem to be discovering another possibility of improvement, and that is...
> 
> I tested some possible scenarios with the tsParticles block in WordPress (1- I loaded the plugins I used to load with my sample; 2- I load the plugins you mentioned with my sample; 3- I load the plugins I used to load with your sample; 4- I load the plugins you mention with your sample)
> 
> Your sample+plugins work in the WordPress editor through the tsParticles block, but it doesn't load on the front-end of the website. This situation only loads correctly in the frontend when I choose "Load tsParticles" instead of "Load tsParticles Slim".
> 
> And there is something happening when choosing "Load Motion" that doesn't happen when choosing other options, but I didn't an extensive test to see if only "Load Motion" is producing a bug. And that is: after saving and reloading the page, the tsParticles block becomes broken - (because I marked the option "Load Motion". If I never choose "Load Motion", when saving and reloading everything, the block works and reloads correctly.
> 
> I hope these investigations and continued feedback helps your work. Sincerely,

Some plugins were missing in the `block.json` file and that creates the issue. It will be fixed in `2.9.1`
